tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -207,7 +207,22 @@ def postgres_ds(context: EphemeralDataContext) -> PostgresDatasource:
 
 
 @pytest.fixture
-def databricks_sql_ds(context: EphemeralDataContext) -> DatabricksSQLDatasource:
+def databricks_creds_populated() -> bool:
```

Comment:
Adding a check/skip logic for `databricks` tests  using the same pattern we are following for `snowflake` tests.